snapshot_tests/shader/base_params_plane_id_5_inputs.scene.json

```diff
@@ -0,0 +1,33 @@
+{
+    "output_id": "output_1",
+    "root": {
+        "type": "shader",
+        "shader_id": "base_params_plane_id",
+        "resolution": {
+            "width": 640,
+            "height": 360
+        },
+        "children": [
+            {
+                "type": "input_stream",
+                "input_id": "input_1"
+            },
+            {
+                "type": "input_stream",
+                "input_id": "input_2"
+            },
+            {
+                "type": "input_stream",
+                "input_id": "input_3"
+            },
+            {
+                "type": "input_stream",
+                "input_id": "input_4"
+            },
+            {
+                "type": "input_stream",
+                "input_id": "input_5"
+            }
+        ]
+    }
+}
```

Comment:
newline

I think there is  an option in vscode to avoid creating files like this